x/vesting/client/cli/tx.go

```diff
@@ -227,3 +227,34 @@ func NewMsgUpdateVestingFunderCmd() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// NewMsgConvertVestingAccount returns a CLI command handler for creating a
+// MsgConvertVestingAccount transaction.
+func NewMsgConvertVestingAccount() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "convert-vesting-account VESTING_ACCOUNT_ADDRESS",
+		Short: "Convert a ClawbackVestingAccount back into a BaseAccount.",
+		Long:  "The ClawbackVestingAccount must have all of it's coins vested.",
```

Comment:
nit, usually this is a longer/ detailed description of the same contents in `Short`